src/lib/components/organizationUsageLimits.svelte

```diff
@@ -0,0 +1,355 @@
+                </Table.Cell>
+                <Table.Cell column="freeLimit" {root}>
+                    <Typography.Text>{formatNumber(freePlanLimits.members)} member</Typography.Text>
+                </Table.Cell>
+                <Table.Cell column="excessUsage" {root}>
+                    {#if isLimitExceeded.members}
+                        <Layout.Stack direction="row" alignItems="center" gap="xs">
+                            <Icon icon={IconArrowUp} size="s" color="--fgcolor-error" />
+                            <Typography.Text color="--fgcolor-error">
+                                {formatNumber(excessUsage.members)} members
+                            </Typography.Text>
+                        </Layout.Stack>
+                    {:else}
+                        <Typography.Text color="--fgcolor-neutral-secondary">
+                            {formatNumber(currentUsage.members)} / {formatNumber(
+                                freePlanLimits.members
+                            )}
+                        </Typography.Text>
+                    {/if}
+                </Table.Cell>
+                <Table.Cell column="manage" {root}></Table.Cell>
+            </Table.Row.Base>
+
+            <!-- Storage Row -->
+            <Table.Row.Base {root}>
+                <Table.Cell column="resource" {root}>
+                    <Typography.Text>Storage</Typography.Text>
+                </Table.Cell>
+                <Table.Cell column="freeLimit" {root}>
+                    <Typography.Text>{freePlanLimits.storage} GB</Typography.Text>
+                </Table.Cell>
+                <Table.Cell column="excessUsage" {root}>
+                    {#if isLimitExceeded.storage}
+                        <Layout.Stack direction="row" alignItems="center" gap="xs">
+                            <Icon icon={IconArrowUp} size="s" color="--fgcolor-error" />
+                            <Typography.Text color="--fgcolor-error">
+                                {excessUsage.storage.toFixed(2)} GB
+                            </Typography.Text>
+                        </Layout.Stack>
+                    {:else}
+                        <Typography.Text color="--fgcolor-neutral-secondary">
+                            {storageUsageGB.toFixed(2)} / {freePlanLimits.storage} GB
+                        </Typography.Text>
+                    {/if}
+                </Table.Cell>
+                <Table.Cell column="manage" {root}></Table.Cell>
+            </Table.Row.Base>
+        </Table.Root>
+    </div>
+</Layout.Stack>
+
+<Modal bind:show={showSelectProject} title="Manage projects" onSubmit={updateSelected}>
+    <svelte:fragment slot="description">
+        Choose which two projects to keep. Projects over the limit will be blocked after this date.
+    </svelte:fragment>
+
+    {#if error}
+        <Alert.Inline status="error" title="Error">{error}</Alert.Inline>
+    {/if}
+
+    <div class="u-overflow-x-auto">
```

Comment:
Only 2 columns in the table root, meaning fixed widths, why the overflow? whats the issue?